packages/fxa-support-panel/lib/server.ts

```diff
@@ -21,7 +21,17 @@ export function init(config: ServerConfig, logger: Logger): hapi.Server {
   const server = new hapi.Server({
     debug: config.env === 'production' ? false : { request: ['error'] },
     host: config.listen.host,
-    port: config.listen.port
+    port: config.listen.port,
+    routes: {
+      security: {
+        hsts: {
+          maxAge: 31536000,
+          includeSubDomains: true,
+          preload: false,
+        },
+        xss: true,
+      },
```

Comment:
Some of these options are here because they are required by the TypeScript interface of that config type.